mutornadomon/monitor.py

```diff
@@ -70,6 +107,12 @@ def __init__(self, host_limit=r'.*', request_filter=LOCALHOST, io_loop=None):
         self._GAUGES = {}
         self._reset_ephemeral()
 
+    def _publish(self):
+        try:
+            self.publisher(self.metrics)
```

Comment:
This resets min/max gauges for the interval, I wonder if we want to enforce the HTTP endpoint *or* this interface, but not both -- otherwise they start stepping on each other.